modules/shasta/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: shasta
+description: The goal of the Shasta long read assembler is to rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input.
+keywords:
+  - nanopore
+  - de-novo
+  - assembly
+  - longread
+tools:
+  - shasta:
+      description: Rapidly produce accurate assembled sequence using as input DNA reads generated by Oxford Nanopore flow cells.
```

Comment:
Can we add a note to say that it isn't deterministic and then we're done.

Reply:
added